Amazon.QLDB.Driver.IntegrationTests/utils/IntegrationTestBase.cs

```diff
@@ -429,4 +436,57 @@ public string GetDisplayName(MethodInfo methodInfo, object[] data)
             return null;
         }
     }
+
+    class ParameterObject
+    {
+        public string Name
+        {
+            get
+            {
+                return Constants.SingleDocumentValue;
+            }
+        }
+
+        public override string ToString()
+        {
+            return "<ParameterObject>{ Name: " + Name + " }";
+        }
+    }
+
+    class ResultObject
+    {
+        public string DocumentId
+        {
+            get; set;
+        }
+
+        public override string ToString()
+        {
+            return "<ResultObject>{ DocumentId: " + DocumentId + " }";
+        }
+    }
+
+    class MySerialization : ISerializer
+    {
+        private static readonly IValueFactory ValueFactory = new ValueFactory();
+
+        public T Deserialize<T>(QLDBSession.Model.ValueHolder v)
+        {
+            return (T)(object)new ResultObject { DocumentId = "Deserialized using custom serializer" };
+        }
+
+        public QLDBSession.Model.ValueHolder Serialize(object o)
```

Comment:
Why are you doing this? Just use the Object Mapper! This is what it is for! :D

Reply:
I believe this was added to test the driver using a custom serializer. 

Although I don't see much value in this test as we already have a test to insert/query c# object using `ObjectSerializer` so I will remove this